lib/absinthe/pipeline.ex

```diff
@@ -92,6 +92,7 @@ defmodule Absinthe.Pipeline do
       # Prepare for Execution
       Phase.Document.CascadeInvalid,
       Phase.Document.Flatten,
+      {Phase.Document.Complexity, options},
```

Comment:
Well, the cost of doing the analysis is very low. It's basically just walking the tree and building an accumulator. It's certainly no worse than any other phase.